frontend/src/components/elements/PlotlyChart/CustomTheme.tsx

```diff
@@ -116,7 +116,6 @@ export function applyStreamlitThemeTemplateLayout(
           color: getGray70(theme),
           size: fontSizes.smPx,
         },
-        standoff: theme.spacing.mdPx,
```

Comment:
Not exactly sure why this is changing but even when I commented out the new changes, the x axis was looking weird...

Reply:
I looked at this core develop streamlit app and ran the st_plotly_chart.py and it looks like my changes are not changing this and this is already on develop.

https://streamlit-core-previews-streamlit-app-develop-preview-tskxqa.streamlit.app/

Reply:
hm, maybe this was caused by the dependency updates?

Reply:
Can you add the missing e2e snapshot to compare the changes?

Reply:
Yep doing that right now

Reply:
I think it was probably caused by dependency updates too